acceptance/openstack/networking/v2/extensions/agents/agents_test.go

```diff
@@ -70,9 +70,6 @@ func TestAgentsRUD(t *testing.T) {
 	th.AssertNoErr(t, err)
 	th.AssertEquals(t, agent.Description, allAgents[0].Description)
 
-	// skip this part
```

Comment:
Any reason you didn't remove the `t.Skip("Skip DHCP agent deletion")` below?

Reply:
I think it's causing problem because there is no DHCP agent when deploying with OVN (the default in recent devstack). Also another issue is that if you remove the agent, later in the tests, you'll have networking issue if you start a VM.
Honestly, this can be re-worked later. But I think it's fine to skip it now like it was the case already.